src/site/apt/tutorial/integration/frameworks.apt

```diff
@@ -39,5 +39,5 @@ Integration with other web Frameworks
 
 
   For a closer look at custom integrations the
-  {{{https://github.com/spring-projects/spring-framework/tree/master/spring-webmvc-tiles3/src/main/java/org/springframework/web/servlet/view/tiles3}spring code}}:
+  {{{https://github.com/spring-projects/spring-framework/tree/v3.2.0.RELEASE/spring-webmvc-tiles3/src/main/java/org/springframework/web/servlet/view/tiles3}spring code}}:
```

Comment:
That may be better, as it would be up to date, but may have problems if they reorganise the repo again. Also the wording about the spring versions would want to be changed to be 3.2 and later I think. If showing the latest is best I can do that.